### PR TITLE
fix: sub menu links should open the correct herf

### PR DESCRIPTION
**Pull request by jared-dickman**
## Instructions

1. PR target branch should be against `main`
2. PR title name should follow this format: https://github.com/mParticle/mparticle-workflows/blob/main/.github/workflows/pr-title-check.yml
3. PR branch prefix should follow this format: https://github.com/mParticle/mparticle-workflows/blob/main/.github/workflows/pr-branch-check-name.yml

## Summary
- this fixes a ts error in overlooked in the previous pr
- allowing links inside of menus to open correctly

## Testing Plan
- [x] Was this tested locally? If not, explain why.
- {explain how this has been tested, and what, if any, additional testing should be done}

## Reference Issue (For mParticle employees only. Ignore if you are an outside contributor)
- Closes https://go.mparticle.com/work/REPLACEME

### CODE REVIEW

File: src/utils/utils.tsx [hunk 1]
```diff
@@ -48,3 +50,16 @@ export function hasImageAtSrc(src: string, hasImageSetter?: (hasImage: boolean)
     img.src = src
   })
 }
+
+export type hrefOptions = { href: string; hrefTarget?: '_self' | '_blank' }
```
Comment (tibuurcio) [inline src/utils/utils.tsx:54]:
```suggestion
export type HrefOptions = { href: string; hrefTarget?: '_self' | '_blank' }
```

Comment (tibuurcio) [reply src/utils/utils.tsx:54]: For consistency

Comment (jared-dickman) [reply src/utils/utils.tsx:54]: or should it be `IHrefOptions`??

Comment (jared-dickman) [reply src/utils/utils.tsx:54]: guess not since its not an interface?

Comment (tibuurcio) [reply src/utils/utils.tsx]: Yeah I mean.. I was never a fan of the `I` prefix even for interfaces. Too much OO for me 😅

**Review by tibuurcio — APPROVED** [review]

File: src/utils/utils.tsx (hunk 1 above)
Comment (jared-dickman) [reply src/utils/utils.tsx]: what does OO stand for??

Comment (jared-dickman) [reply src/utils/utils.tsx]: let the record show, i am always a fan of less typing, as long as it doesnt obscure the meaning

**Pull request merged**